extension/app/ts/background/background.ts

```diff
@@ -428,19 +433,23 @@ async function onContentScriptConnected(port: browser.runtime.Port) {
 		if (!window.interceptor.websiteTabIcons.has(tabId)) {
 			updateExtensionIcon(port)
 			port.onDisconnect.addListener(() => {
-				updateExtensionIcon(port)
```

Comment:
Why aren't we doing this anymore?

Reply:
this is because the port is closed, which means that the tab is closed, no need to update extension icon for such tab as it doesn't exist anymore